File: C/2808.c
#include <stdio.h>
#include <stdlib.h>

int main(){

    char a[3], b[3];

    scanf("%s %s", a, b);

    int li, lf, ni, nf;

    li = (int) a[0];
    ni = (int) a[1];
    lf = (int) b[0];
    nf = (int) b[1];

    if((abs(lf-li) == 1 && abs(nf - ni) == 2) || (abs(lf-li) == 2 && abs(nf - ni) == 1)){
        printf("VALIDO\n");
    }else{
        printf("INVALIDO\n");
    }

    return 0;
}

File: Python/2808.py
posi, posf = input().split()

posiLetra, posiNum = ord(posi[0]), int(posi[1])
posfLetra, posfNum = ord(posf[0]), int(posf[1])

if (abs(posfLetra - posiLetra) == 1 and abs(posfNum - posiNum) == 2) or (abs(posfLetra - posiLetra) == 2 and abs(posfNum - posiNum) == 1):
    print("VALIDO")
else:
    print("INVALIDO")